modules/memo/src/main/Picfit.scala

```diff
@@ -94,7 +94,7 @@ final class PicfitApi(coll: Coll, val url: PicfitUrl, ws: StandaloneWSClient, co
     def store(image: PicfitImage, part: SourcePart): Funit =
       ws
         .url(s"${config.endpointPost}/upload")
-        .post(Source(part.copy[ByteSource](filename = image.id.value, key = "data") :: List()))(using
+        .post(Source(part.copy[ByteSource](filename = image.id.value, key = "data") :: Nil))(using
```

Comment:
This seems more natural to me; but could also `.post(Source(List(...)))` if I read it right -- I can't tell the parens at a glance, so `Source(List(...))` is structurally more obvious.